### PR TITLE
more cdf tweaks

### PR DESCRIPTION
**Pull request by moldover**
Tear it up...

### CODE REVIEW

File: src/main/java/com/rcv/ResultsWriter.java [hunk 1]
```diff
@@ -702,10 +702,13 @@ void generateCdfJson(List<CastVoteRecord> castVoteRecords)
...
           entry("Value", config.getNameForCandidateCode(candidateCode))
       );
+      // codeMap is wrapped in an array
+      ArrayList<Map<String, String>> codeArray = new ArrayList<>();
+      codeArray.add(codeMap);
       contestSelections.add(Map.ofEntries(
           entry("@id", getCdfIdForCandidateCode(candidateCode)),
           entry("@type", "CVR.ContestSelection"),
-          entry("Code", codeMap)
+          entry("Code", codeArray)
```
Comment (tarheel) [inline src/main/java/com/rcv/ResultsWriter.java]: this is simpler and matches what I've done elsewhere:

`entry("Code", new Map[]{codeMap}),`

File: src/main/java/com/rcv/ResultsWriter.java [hunk 2]
```diff
@@ -681,7 +681,7 @@ void generateCdfJson(List<CastVoteRecord> castVoteRecords)
     Map<String, Object> contestMap =
         Map.ofEntries(
             entry("ContestId", CDF_CONTEST_ID),
-            entry("CVRContestSelection", selectionMapList),
+            entry("ContestSelection", selectionMapList),
```
Comment (tarheel) [inline src/main/java/com/rcv/ResultsWriter.java]: I couldn't tell for sure from John's email whether this was the thing he wanted us to change.

Comment (moldover) [reply src/main/java/com/rcv/ResultsWriter.java]: Yeah, his email (and the standard) are confusing, but I think it's right.  Looking at all the other @type elements it checks out.

File: src/main/java/com/rcv/ResultsWriter.java (hunk 1 above)
Comment (moldover) [reply src/main/java/com/rcv/ResultsWriter.java]: 👍

**Pull request merged**